src/main/java/com/puppycrawl/tools/checkstyle/checks/design/InnerTypeLastCheck.java

```diff
@@ -36,6 +36,28 @@
  * <pre>
  * &lt;module name=&quot;InnerTypeLast&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class Test {
+ *   public void test1() {} // ok
+ *   class InnerTest1 {}
+ *   private String s; // violation, field should be declared before inner classes.
+ *
+ *   class InnerTest2 {
+ *     public void innerTest() {
+ *       int n; // ok
+ *       class LocalClass {
+ *         public void test3() {} // ok
+ *       }
+ *       int x; // ok
+ *     }
+ *   }
```

Comment:
instead of nested ok examples, please make another class with ok case (sibling of Test), e.g.
```
class Test {
 // not ok code
}

class Test2 {
 // ok code
}

Reply:
Done. Please check if it's correct or not and let me know.